org.gnome.Shotwell.json

```diff
@@ -0,0 +1,175 @@
+            "cleanup": ["/share/gtk-doc"],
+            "config-opts": [ "--disable-always-build-tests",
+                "--disable-introspection",
+                "--disable-Werror",
+                "--disable-static",
+                "--disable-goa" ],
+                "sources": [
+                    {
+                        "type": "archive",
+                        "url": "https://download.gnome.org/sources/libgdata/0.17/libgdata-0.17.9.tar.xz",
+                        "sha256": "85c4f7674c0098ffaf060ae01b6b832cb277b3673d54ace3bdedaad6b127453a"
+                    }
+                ]
+        },
+
+
+        {
+            "name": "exiv2",
+            "cleanup": [ "/bin" ],
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/Exiv2/exiv2/",
+                    "commit" : "e7ffd83af29187190da7f98dcbca8a4d70e19582"
+                }
+            ]
+        },
+        {
+            "name": "gexiv2",
+            "cleanup" : ["/lib/girepository-1.0", "/share/gir-1.0"],
+            "config-opts" : ["--without-python2-girdir", "--without-python3-girdir"],
+            "build-options" : {
+                "env": {
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+                }
+            },
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/gexiv2.git/"
+                }
+            ]
+        },
+
+        {
+            "name": "libraw",
+            "config-opts": [ "--disable-examples", "--disable-static" ],
+            "cleanup": [ "/share/doc" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.libraw.org/data/LibRaw-0.18.13.tar.gz",
+                    "sha256": "cb1f9d0d1fabc8967d501d95c05d2b53d97a2b917345c66553b1abbea06757ca"
+                },
+                {
+                    "type": "patch",
+                    "path": "libraw-pkgconfig.patch"
+                }
+            ]
+        },
+        {
+            "name": "shotwell",
+            "buildsystem": "meson",
+            "config-opts" : ["-Dudev=false", "-Dinstall-apport-hook=false", "-Dface-detection=false"],
+            "sources" : [
+                {
+                    "type": "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/shotwell",
+                    "commit" : "2d5cf607734bd58d86804d98307e386058785103"
+                },
+                {
+                    "type" : "patch",
+                    "path" : "shotwell-desktop.patch"
+                }
+            ],
+            "post-install": [
+                "glib-compile-schemas /app/share/glib-2.0/schemas/"
```

Comment:
This should probably be handled upstream.

Reply:
I am also not sure why I need(ed) this

Reply:
If I do not do this, settings are not persisted - I'm not sure why that is happening, though. Also I'm pretty sure I read somewhere that this should happen in the flatpak stack?

Reply:
Ah wait, you mean I should handle that in a meson post-install in the shotwell source?